recipe/meta.yaml

```
@@ -1,25 +1,27 @@
 {% set name = "arraymap" %}
-{% set version = "0.2.2" %}
+{% set version = "0.3.0" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 8409b499fb2cb3fdd253e85c5277828c9215a9692235dad202069f0384ab5f8f
+  sha256: 8a303d035f42038429b18d901828389ffee468efed991eb1c5e500508ed4981b
 
 build:
   number: 0
+  skip: true  # [py<39]  
   script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   build:
     - {{ compiler('c') }}
+    - {{ stdlib("c") }}
   host:
     - python
     - pip
-    - numpy
+    - numpy >=2.0.0
```

```suggestion
    - numpy
```

Revert this please. Instead place a copy of the [migration file](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/migrations/numpy2.yaml) under `.ci_support/migrations/` and rerender.